klippy/kinematics/delta.py

```diff
@@ -30,6 +30,8 @@ def __init__(self, toolhead, config):
         self.max_z_velocity = config.getfloat(
             'max_z_velocity', self.max_velocity,
             above=0., maxval=self.max_velocity)
+        self.max_z_accel = config.getfloat('max_z_accel', self.max_accel,
```

Comment:
I'm sorry, but this appears to be unused (maybe the code was lost during rebase/merge)?

Reply:
how do you mean unused..

Reply:
removed

Reply:
Sorry, I meant it the other way around: you created this member variable (self.max_z_accel), which is good, but the code to actually make use of this variable was accidentally removed (it seems). Now if I look at the diffs:
https://github.com/Klipper3d/klipper/pull/4689/files
I see that only documentation is updated. But the actual code to limit Z accel on Delta is gone. Please add it back :)

Reply:
sorry I'm new at this.. =)

Reply:
think I got it right.